src/sdk/nsfs_glacier_backend/backend.js

```diff
@@ -271,6 +303,29 @@ class GlacierBackend {
 
         return restore_status.state === GlacierBackend.RESTORE_STATUS_ONGOING;
     }
+
+	/**
+	 * storage_class_from_xattr returns a parsed storage class derived from the given
+	 * extended attribute. It will use DMAPI EA if `use_dmapi` is set to true.
+	 *
+	 * NOTE: For `use_dmapi` to work, the xattr must have been retrieved using fs_context
+	 * where backend is set to 'GPFS'.
+	 *
+	 * @param {nb.NativeFSXattr} xattr
+	 * @param {Boolean} [use_dmapi]
+	 * @returns {nb.StorageClass}
+	 */
+	static storage_class_from_xattr(xattr, use_dmapi) {
+		if (
+            use_dmapi &&
```

Comment:
I think you can simply use `config.NSFS_GLACIER_USE_DMAPI` inside here and avoid passing it in which will also reduce the clutter in calling this function.
```suggestion
            config.NSFS_GLACIER_USE_DMAPI &&
```

Reply:
Using global configs makes testing quite tricky that's why I deliberately added it as a parameter wherever I could.

Reply:
ok, so maybe just set this config as the default for the argument so you can still override for testing but don't need to pass in normal calls
```js
static storage_class_from_xattr(xattr, use_dmapi = config.NSFS_GLACIER_USE_DMAPI) {
```